### ISSUE TITLE
Errors!

### ISSUE THREAD

**Issue by LeoL2312** [gloglas/rpi_gui#3]
I am trying to install but getting these errors

root@kali:~/rpi_gui# sh setup.sh
Hit:1 http://mirror.serverius.net/kali kali-rolling InRelease
Reading package lists... Done
Reading package lists... Done
Building dependency tree
Reading state information... Done
**E: Unable to locate package python3-netifaces**
Requirement already satisfied: spidev in /usr/local/lib/python3.10/dist-packages (3.5)
Requirement already satisfied: keyboard in /usr/local/lib/python3.10/dist-packages (0.13.5)
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
**Setting up interfaces
: Invalid argumentt create /boot/config.txt
: Invalid argumentt create /boot/config.txt
: Invalid argumentt create /boot/config.txt
: not found0:
-e**
Installing munifying

fatal: destination path 'munifying' already exists and is not an empty directory.
setup.sh: 13: cd: can't cd to munifying
: not found4: ./install_libusb.sh
can't load package: package .: no Go files in /root/rpi_gui
-e
Munifying Failed!!!

: not found6:
-e
Setting up startup scripts

: not found2:
Invalid unit name "ctrl-alt-del.target
" escaped as "ctrl-alt-del.target\x0d" (maybe you should use systemd-escape?).
Unknown operation daemon-reload
.
: not found5:
mkdir: cannot create directory 'rpi_gui/log\r': No such file or directory
mkdir: cannot create directory 'rpi_gui/ducky\r': No such file or directory
: not found8:
-e
Now you need to reboot!

**Comment by gloglas** [comment]
Are you running on this image for RPI 0W v1?
https://github.com/RoganDawes/P4wnP1_aloa/releases/tag/v0.1.1-beta

**Comment by gloglas** [comment]
Try installing netifaces with pip instead.
And also bit of a mistake on my part. You need to copy and run setup from /root directory.

**Comment by LeoL2312** [comment]
> 

yes I am using rpi 0w v1.1

**Comment by dseminoff** [comment]
I am running into the same issues. Are you still available on this project?

**Comment by gloglas** [comment]
Nope. Try setup doing manually, perhaps running it in bash or consult AI.
